### PR TITLE
Textual improvements

### PR DESCRIPTION
**Pull request by r00ster91**
This fixes more typos and does some other minor textual improvements. It for example makes the `BigDecimal` documentation visible in the API docs.

### CODE REVIEW

File: src/file/info.cr [hunk 1]
```diff
@@ -75,8 +75,7 @@ class File
     end
   end
 
-  # A `File::Info` contains metadata regarding a file. It is returned by
-  # `File.info`, and `File#info`.
+  # A `File::Info` contains metadata regarding a file. It is returned by `File.info`.
```
Comment (RX14) [inline src/file/info.cr]: Why was this changed?

Comment (r00ster91) [reply src/file/info.cr]: Hmm I probably thought for some reason that the method doesn't exist.

File: src/big/big_decimal.cr [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
   def initialize(big_decimal_str : String, reason : String)
     super("Invalid BigDecimal: #{big_decimal_str} (#{reason})")
   end
 end
 
+# A `BigDecimal` can represent arbitrarily large precision decimals.
+#
+# It is internally represented by a pair of `BigInt` and `UInt64`: value and scale.
+# Value contains the actual value, and scale tells the decimal point place.
+# E.g. when value is 1234 and scale 2, the result is 12.34.
```
Comment (straight-shoota) [inline src/big/big_decimal.cr]:
```suggestion
# E.g. when value is `1234` and scale `2`, the result is `12.34`.
```
